src/App.js

```diff
@@ -24,10 +27,12 @@ const routes = [
     createRoute("/", <div><LandingPage /><a href='https://www.freepik.com/vectors/people' style={{display:"none"}}>People vector created by pikisuperstar - www.freepik.com</a></div>),
     createRoute("/interview/:interviewId/:companyId", <Meeting />, {}),
     createRoute("/interview/:interviewId/", <Meeting />, {}),
-    createRoute("/offers/add", <OfferForm />),
     createRoute("/offers/apply/:id", <ApplyForm />),
-    createRoute("/offers", <AllOffersView />),
-    createRoute("/offers/:id", <AllOffersView />),
+    createRoute("/offers", <OffersView getOffers={offersAPI.getAllOffers}/>),
+    createRoute("/offers/:id", <OffersView getOffers={offersAPI.getAllOffers}/>),
+    createRoute("/hr/offers", <OffersView getOffers={() => offersAPI.getOffersFromHr()} buttons={hrOfferButtons}/>),
+    createRoute("/hr/offers/add", <OfferForm onSubmit={(form) => offersAPI.create(form)}/>),
+    createRoute("/hr/offers/edit/:id", <OfferForm onSubmit={(form) => offersAPI.update(form)}/>),
```

Comment:
Nie wiem czy to jakaś sensowna uwaga i nie wiem czy tak się da, ale jakoś bardziej pasowałaby mi kolejność offers/:id/edit. Ale to nie ma aż takiego znaczenia